src/cmake/AxomMacros.cmake

```diff
@@ -115,6 +115,46 @@ macro(axom_add_component)
 endmacro(axom_add_component)
 
 
+##------------------------------------------------------------------------------
+## axom_add_test(NAME            [name]
+##               COMMAND         [command] 
+##               NUM_MPI_TASKS   [n]
+##               NUM_OMP_THREADS [n]
+##               CONFIGURATIONS  [config1 [config2...]])
+##
+## Wrapper around axom_add_test() that handles functionality that Axom applies to all
```

Comment:
Should be "around blt_add_test() that handles functionality"

Reply:
yepp, missed this after the sed script. Thanks!